google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfiguration.java

```diff
@@ -99,6 +118,16 @@ public String getModuleName() {
     return moduleName;
   }
 
+  @Attribute("appYamlPath")
+  public String getAppYamlPath() {
```

Comment:
appYaml, and Dockerfile weren't previously part of the persisted configuration, correct?

I remember instead using moduleName as a the persisted key which can be used as a pointer into the facet configuration. How do these work together now?